swap_meet/clothing.py

```diff
@@ -0,0 +1,9 @@
+from .item import Item
+
+class Clothing(Item):
+    def __init__(self, age = None, category = "Clothing", condition = 0):
+        super().__init__(age, category, condition)
```

Comment:
Nice! Another option is to remove `category` from the instance constructor and move it to the parent constructor (aka super() constructor). This will remove any chance of a clothing item from becoming a different category. Remind me to demonstrate during our 1:1. See below:

```suggestion
    def __init__(self, age = None, condition = 0):
        super().__init__(age, category="Clothing", condition)
```

This same logic can be used on `Decor` and `Electronics`.